tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "target": "es2015",
+    "module": "es2015",
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": false,
+    "moduleResolution": "node",
+    "stripInternal": true,
+    "preserveConstEnums": true,
+    "listFiles": true,
+    "declaration": true,
+    "removeComments": true,
+    "lib": ["es2015", "dom"]
```

Comment:
Thanks, i suspected, that it must be smth from beta, but didn't find the relevant information quickly :)